pintracker/statelesstracker/config.go

```diff
@@ -0,0 +1,86 @@
+package statelesstracker
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/ipfs/ipfs-cluster/config"
+)
+
+const configKey = "statelesstracker"
```

Comment:
`stateless` works, I think. Note that it already goes inside a `pin_tracker` config section.